packages/grid/_modules_/grid/models/api/gridFilterApi.ts

```diff
@@ -20,12 +20,6 @@ export interface GridFilterApi {
    * @param {GridFilterItem} item The filter to update.
    */
   upsertFilter: (item: GridFilterItem) => void;
-  /**
-   * Applies a [[GridFilterItem]] on alls rows. If no `linkOperator` is given, the "and" operator is used.
-   * @param {GridFilterItem} item The filter to be applied.
-   * @param {GridLinkOperator} linkOperator The link operator to use.
-   */
-  applyFilter: (item: GridFilterItem, linkOperator?: GridLinkOperator) => void;
```

Comment:
This makes the PR a breaking change. I've updated the description to provide an alternative.